src/test/java/com/google/api/codegen/testdata/php_client_impl_library.baseline

```diff
@@ -645,7 +570,7 @@ class LibraryServiceGapicClient
      * ```
      * try {
      *     $libraryServiceClient = new LibraryServiceClient();
-     *     $formattedName = LibraryServiceClient::formatShelfName("[SHELF_ID]");
+     *     $formattedName = $this->shelfPath("[SHELF_ID]");
```

Comment:
This was copying Ruby, based on feedback from DPE (see https://github.com/googleapis/toolkit/blob/master/src/main/java/com/google/api/codegen/transformer/ruby/RubySurfaceNamer.java#L82)

I like "name" as well, although I don't have strong feelings between "name" and "path" - I think "path" is descriptive in a way that "name" isn't.

@bshaffer @dwsupplee @jdpedrie WDYT?